app/controllers/pbs/events_controller.rb

```diff
@@ -72,6 +73,22 @@ def assign_contact_attrs_with_supercamp_flag
     contact_attrs
   end
 
+  def preview_camp_application_validation
+    return unless entry.is_a?(Event::Campy)
+    return unless [entry.leader, entry.coach].include? current_person
+    return if entry.camp_submitted?
+
+    original_camp_submitted_at = entry.camp_submitted_at
+    entry.camp_submitted_at = Time.zone.now.to_date
+    if entry.valid?
+      flash.now[:notice] = "#{I18n.t('events.create_camp_application.flash.preview_success')}"
+    else
+      flash.now[:warn] = "#{I18n.t('events.create_camp_application.flash.preview')}" \
+                        "<br />#{entry.errors.full_messages.join('; ')}"
+    end
+    entry.camp_submitted_at = original_camp_submitted_at
```

Comment:
Zweitteilige Suggestion: Man kann AR nutzen, um das Attribute wieder zurückzusetzen. (1/2)

Und kann hier AR selbst darum bitten:
```suggestion
    entry.restore_attributes # or entry.restore_attributes([:camp_submitted_at])
```

Siehe: https://devdocs.io/rails~6.1/activemodel/dirty#method-i-restore_attributes